gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4-bin.zip
```

Comment:
I don't see any binary changes even though you increased gradle version?

Reply:
Thanks. I will use `gradle wrapper` to upgrade the Gradle version.

Reply:
Have used `./gradlew wrapper --gradle-version 6.4` command to upgrade.